metrics/lib/features/dashboard/presentation/strings/dashboard_strings.dart

```diff
@@ -0,0 +1,11 @@
+class DashboardStrings {
```

Comment:
Great job so far! I think it would be great to add documentation here, for class & method?